test/src/helpers.cc

```diff
@@ -78,6 +103,18 @@ void check_partitions(
     CHECK(unsplittable);
   } else {
     CHECK(!unsplittable);
+#if 0  //&& DIAGNOSING
```

Comment:
Is this needed?

Reply:
ch8931 created, comments and diagnostic code removed